docker/Dockerfile.gpu

```diff
@@ -2,10 +2,10 @@ FROM nvidia/cuda:9.0-cudnn7-devel
 
 COPY install.sh /
 
-RUN /install.sh
+RUN /install.sh && pip install mxnet-cu90 && mkdir /mxnet_model_server
 
-RUN pip install mxnet-cu90
+COPY wsgi.py setup_mms.py mxnet-model-server.sh /mxnet_model_server/
 
-ADD mms_docker_gpu mms_docker_gpu
+ENV PATH="/mxnet_model_server:${PATH}" GPU_IMAGE=1 gpu_id=0
```

Comment:
GPU_IMAGE -> MXNET_MODEL_SERVER_GPU_IMAGE

Reply:
Changed this and the usage of this environment variable.